qemu/qemu_arch_base.go

```diff
@@ -0,0 +1,87 @@
+// +build !s390x
```

Comment:
We need to add the license headers here.  As far as I can tell all the code in this file and in qemu_arch_base_test.go is entirely copied from existing code found in qemu.go and qemu_test.go so it's appropriate to copy the license header from these files.

Reply:
@markdryan thanks! Ok then I'll add the license headers as in the other files.